### DISCUSSION TITLE
Count-up without trailing minus

### DISCUSSION THREAD

**Discussion by michaellammers** [cpvalente/ontime#1375] [Q&A] [answered]
I have been playing with the count-up, but can't manage to start at 00:00:00 and end at 01:00:00 for example. In all my efforts the counter counts down or counts up with a minus sign in front. Although I can certainly understand if this use is not within scope, I was just wondering if this was possible.
Thanks for this great tool!

**Comment by alex-Arc** [comment]
Hey @michaellammers 
Could you share a screenshot of the problem

**Comment by michaellammers** [comment]
Sure @alex-Arc, I've attached a screenshot of the ontime interface with a python monitor (forget the 'volume' indexation, it was just for testing) and the web monitor. After my initial post I checked the web monitor an saw that that one does count up, but in a previous attempt I only looked at the main ontime screen with the big clock on the left counting down, no matter what settings I use. So, although the web monitor counts up, both the python monitor and the main clock on the left count up. But I am not sure if this is intentional and expected behavior?

![Scherm­afbeelding 2024-12-17 om 07 37 25](https://github.com/user-attachments/assets/844e0992-3abb-4b32-97ed-4dfbc6c0b6bd)

**Reply by alex-Arc** [reply, marked as answer]
Ahh I see
Yes, internally ontime will always count down the duration, that is the `current` value that you are most likely getting in python 
that is the same value that all Operator displays use 
the timer type is purely a display setting for timer views
(we are currently looking at how to make this more clear, add looking for feedback in the [discord server](https://discord.gg/QDj5rRaE) )
if you want the same value output to integrations you can use the elapsed value `{{human.elapsedTime}}` or  `{{timer.elapsed}}`

**Reply by michaellammers** [reply]
All clear! Thanks for the explanation Alex! 🙏
